packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/dateRangeBox.tests.js

```diff
@@ -40,6 +40,7 @@ const CALENDAR_CELL_CLASS = 'dx-calendar-cell';
 const CALENDAR_CONTOURED_CELL_CLASS = 'dx-calendar-contoured-date';
 const APPLY_BUTTON_SELECTOR = '.dx-popup-done.dx-button';
 const CANCEL_BUTTON_SELECTOR = '.dx-popup-cancel.dx-button';
+const CALENDAR_CELL_IN_RANGE_CLASS = 'dx-calendar-cell-in-range';
```

Comment:
this test is related to dateRangeBox strategy, let's move it to `dateRangeBox.strategy.tests.js`

Reply:
[fbe8a22](https://github.com/DevExpress/DevExtreme/pull/28423/commits/fbe8a2281282c096308a8bf5bde0fa5f548498d5)

Reply:
Alex, the fix is applied for Calendar, so I believe the test should be for calendar itself [f7f9431](https://github.com/DevExpress/DevExtreme/pull/28423/commits/f7f94318a5cdfd609fae68652923486c18127dcf)